packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -48,6 +48,12 @@ export interface ITelemetryLoggerPropertyBags{
     error?: ITelemetryLoggerPropertyBag;
 }
 
+export type TelemetryEventTypes =
+    | ITelemetryBaseEvent
+    | ITelemetryGenericEvent
+    | ITelemetryErrorEvent
+    | ITelemetryPerformanceEvent;
```

Comment:
I think: These can move to Telemetry-utils as well with `Ext` suffix or whatever, they are the existing "internal" event types that all our code uses when logging.  The `ITelemetryLogger` interface should be updated to use the new types instead of these.